packages/one-app-bundler/webpack/plugins/restrict-runtime-symbols.js

```diff
@@ -0,0 +1,34 @@
+import fs from 'node:fs';
+import {
+  validateAppConfig,
+} from '@americanexpress/one-app-dev-bundler';
+
+class RestrictRuntimeSymbols {
+  // eslint-disable-next-line class-methods-use-this -- no need for this
+  apply(compiler) {
```

Comment:
Where is compiler coming from and is this just passing the ```this``` value of compiler to apply?

Reply:
compiler is being passed in from webpack.

I have a sneaking suspicion what is happening here is we are making a class that looks like a function. But this is how the plugin I referenced to make this did it. So i just kept this done the same way as our other plugin

Reply:
funky stuff